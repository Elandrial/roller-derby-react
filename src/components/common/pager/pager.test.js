import Pager from './pager';
import renderer from 'react-test-renderer';

describe('Pager component',() => {
  it('renders correctly', () =>{
    const tree = renderer
    .create(<Pager pages={9} currentPage={1} ariaName="Events" maxMainButtons={5} maxSideButtonsMD={1} maxSideButtonsLG={2}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  })
});