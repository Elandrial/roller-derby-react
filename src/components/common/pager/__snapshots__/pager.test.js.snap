// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pager component renders correctly 1`] = `
<div
  className="page-control"
>
  <button
    aria-label="Go to previous Events page"
    disabled={true}
    onClick={[Function]}
  >
    &lt;
  </button>
  <button
    aria-current="true"
    aria-label="Current Events Page, Page 1"
    className=""
    disabled={true}
    onClick={[Function]}
  >
    1
  </button>
  <button
    aria-current="true"
    aria-label="Go to Events page 2"
    className=""
    disabled={false}
    onClick={[Function]}
  >
    2
  </button>
  <button
    aria-current="true"
    aria-label="Go to Events page 3"
    className=""
    disabled={false}
    onClick={[Function]}
  >
    3
  </button>
  <button
    aria-current="true"
    aria-label="Go to Events page 4"
    className="hidden-m hidden-s hidden-xs"
    disabled={false}
    onClick={[Function]}
  >
    4
  </button>
  <button
    aria-current="true"
    aria-label="Go to Events page 5"
    className="hidden-l hidden-m hidden-s hidden-xs"
    disabled={false}
    onClick={[Function]}
  >
    5
  </button>
  <button
    aria-label="Go to next Events page"
    disabled={false}
    onClick={[Function]}
  >
    &gt;
  </button>
</div>
`;
